docker/jenkins/rstudio-version.sh

```diff
@@ -59,7 +59,12 @@ if [[ $DEBUG = false ]]; then
     EXTRA_CP_ARGS=--quiet
 fi
 
-MAJOR_VERSION="$(date -u '+%Y-%m')"
+if [ -e "RELEASE_VERSION" ]; then
```

Comment:
This makes the assumption that the script is running in the root directory of the repo, which is indeed true when we call it in Jenkins but might not always be true (and we say at the top that you can run it manually). I think we should either have the script infer the repo root and look there for `RELEASE_VERSION`, or have the release version taken as an optional parameter (that we read in Jenkins and then pass into the script).

Reply:
The reason I treated it this way is becahse we have prior art in the file (https://github.com/rstudio/rstudio/pull/9695/files#diff-be5e186787a1b8e696089d63807242f9cb48b658b42cc24db5cd54f9fd260775R74-R77). Would you like me to change both?